CITATION.bib

```diff
@@ -1,6 +1,8 @@
-@misc{Qiskit,
-    author = {{Qiskit contributors}},
-    title = {Qiskit: An Open-source Framework for Quantum Computing},
-    year = {2023},
-    doi = {10.5281/zenodo.2573505}
+@misc{javadiabhari2024quantum,
```

Comment:
Maybe keep the same citekey?
```suggestion
@misc{Qiskit,
```

Reply:
Also, I saw `techreport` used for this kind of white papers. Not sure if it makes any difference.

Reply:
should probably be at least `Qiskit2024` or something by default - it's a pretty non-zero chance that "the Qiskit developers" will publish another paper at some point.

Reply:
I went with `qiskit2024` for simplicity.